build/bootstrap/bootstrap-debian.sh

```diff
@@ -34,11 +34,6 @@ curl -fsSL https://get.pnpm.io/install.sh | env PNPM_VERSION=8.6.6 sh -
 
 sudo adduser "${USER}" docker
 
-# Configure environment variables.
-echo 'export PATH="${PATH}:$HOME/go/src/github.com/cockroachdb/cockroach/bin:/usr/local/go/bin"' >> ~/.bashrc_bootstrap
-echo '. ~/.bashrc_bootstrap' >> ~/.bashrc
-. ~/.bashrc_bootstrap
```

Comment:
Make sure to rebase given #139469 that modified this code. You'll want to incorporate that change in this PR too.

Reply:
Yeah, there are expected merge conflicts, I will resolve them.

Reply:
Rebased and fixed merge conflicts.